### ISSUE TITLE
Browser not Supported

### ISSUE THREAD

**Issue by nirajan-pant** [hatnote/montage#156]
Montage is not working in Firefox, Brave, Microsoft Edge. Only header strip with MONTAGE is showing.

**Comment by mahmoud** [comment]
Hmm, that doesn't sound good, thanks for reporting. Now, Firefox is my primary browser, and I'm seeing Montage work fine, I just checked here: https://tools.wmflabs.org/montage/#/

I expect it to work fine on Brave as well, can't promise on IE, depending on version.

So, on which URL are you seeing this behavior, have you tried clearing caches, and logging out (in the header) and back in again?

If none of that works, can you pop open the dev tools and see if you're getting any error responses?

**Comment by atsirlin** [comment]
It's working fine in all these browsers. You should just wait for some time until the rest of the page is loaded and make sure that your internet connection is fast enough.

**Comment by nirajan-pant** [comment]
I am trying to browse
https://tools.wmflabs.org/montage/#/campaign/278-wiki-loves-monuments-2019-nepal
but its not opening. I have waited for half an hour. I think it should be redirected to login page or display some error message.

I checked by browsing https://tools.wmflabs.org/montage/#/
it worked.

**Comment by atsirlin** [comment]
Yes, it does not work when you are not logged in, and it should probably display some message instead of being silent.
